package/kedro_viz/models/flowchart.py

```diff
@@ -411,6 +411,8 @@ def __post_init__(self, task_node: TaskNode):
         if kedro_node._name is not None:
             self.run_command = (
                 f"kedro run --to-nodes={task_node.namespace}.{kedro_node._name}"
+                if task_node.namespace is not None
+                else f"kedro run --to-nodes={kedro_node._name}"
```

Comment:
Is this the way we normally write `if/else` statements in the codebase?

Reply:
Yes we do use this style.